src/basic/barrier.h

```diff
@@ -70,19 +70,20 @@ bool barrier_sync_next(Barrier *b);
 bool barrier_sync(Barrier *b);
 
 static inline bool barrier_i_aborted(Barrier *b) {
-        return b->barriers == BARRIER_I_ABORTED || b->barriers == BARRIER_WE_ABORTED;
+        return IN_SET(b->barriers, BARRIER_I_ABORTED, BARRIER_WE_ABORTED);
 }
 
 static inline bool barrier_they_aborted(Barrier *b) {
-        return b->barriers == BARRIER_THEY_ABORTED || b->barriers == BARRIER_WE_ABORTED;
+        return IN_SET(b->barriers, BARRIER_THEY_ABORTED, BARRIER_WE_ABORTED);
 }
 
 static inline bool barrier_we_aborted(Barrier *b) {
         return b->barriers == BARRIER_WE_ABORTED;
 }
 
 static inline bool barrier_is_aborted(Barrier *b) {
-        return b->barriers == BARRIER_I_ABORTED || b->barriers == BARRIER_THEY_ABORTED || b->barriers == BARRIER_WE_ABORTED;
+        return IN_SET(b->barriers, BARRIER_I_ABORTED, BARRIER_THEY_ABORTED,
+                      BARRIER_WE_ABORTED);
```

Comment:
I build my own coccinelle rpm with the following "patch": `sed -r -i 's/max_width = 78/max_width = 200/' parsing_c/flag_parsing_c.ml`. We get such ugly wrapping otherwise.